specs/darwin/wifi_networks.table

```diff
@@ -0,0 +1,17 @@
+table_name("wifi_networks")
+description("OS X known/remembered Wi-Fi networks list.")
+schema([
+    Column("ssid", TEXT, "SSID octets of the network"),
+    Column("ssid_string", TEXT, "SSID String of the network"),
```

Comment:
I know this is what everyone refers to as the SSID and I believe this is the actual 32-octect SSID value, or maybe this is an OS X alias the user or machine can assign? Are there instances where the SSID key is different that SSIDString? Maybe we call `ssid_string` = `network_name`?